src/main/kotlin/nl/avisi/structurizr/site/generatr/GenerateSiteCommand.kt

```diff
@@ -97,7 +97,8 @@ class GenerateSiteCommand : Subcommand(
                 createStructurizrWorkspace(workspaceFileInRepo)
                 true
             } catch (e: Exception) {
-                println("Bad Branch $branch")
+                errorMessage = e.message ?: "Unknown error"
```

Comment:
This does not compile. This should:
```suggestion
                val errorMessage = e.message ?: "Unknown error"
```

Reply:
Oh got it, not familiar with Kotlin, my apologies :)